Resources/Prototypes/Entities/Clothing/Belt/belts.yml

```diff
@@ -409,12 +409,17 @@
     state: sheath
   - type: Clothing
     sprite: Clothing/Belt/sheath.rsi
-  - type: Storage
-    grid:
-    - 0,0,1,1
-    whitelist:
-      tags:
-        - CaptainSabre
+  - type: Item
+    size: Ginormous
+  - type: ItemSlots
+    slots:
+      item:
+        name: Sabre
+        insertVerbText: Sheathe
+        ejectVerbText: Unsheathe
```

Comment:
localize this with .ftl files.

https://docs.spacestation14.com/en/general-development/codebase-info/conventions.html?highlight=local#localization

Reply:
Done.